### DISCUSSION TITLE
BarrierD command line?

### DISCUSSION THREAD

**Discussion by noharminthat** [debauchee/barrier#1851] [Q&A]
I'm trying to stop/start Barrier from the command line. BarrierS says the start/stop commands have been deprecated and to use BarrierD instead. However, running BarrierD gives me an error "The service process could not connect to the service controller". --help gives the same error. I can't find any information on BarrierD in the wiki. Anybody know what the options are for BarrierD? Thanks

**Comment by cptfinch** [comment]
i have the same question please. Where is info on barrierd ?
